app/bundles/CampaignBundle/Entity/CampaignRepository.php

```diff
@@ -501,7 +501,7 @@ public function getCampaignLeadCount($campaignId, $leadId = null, $pendingEvents
                 $q->getSQL(),
                 $q->getParameters(),
                 $q->getParameterTypes(),
-                new QueryCacheProfile(600, __METHOD__)
+                new QueryCacheProfile(600, 'campaign_lead_count_'.md5($q->getSQL()))
```

Comment:
@escopecz this will cause an issue when we upgrade, as we are making the profile name with the same name all the time. we need to make it unique

Reply:
It looks like we don't need to provide the second parameter at all and Doctrine will generate it automatically. See

https://github.com/doctrine/dbal/blob/3.6.x/src/Cache/QueryCacheProfile.php#L130

and

https://github.com/doctrine/dbal/blob/4d27b9844749d611a963ea0d3caa4a9aec3fc1bb/src/Connection.php#L1132

Reply:
Solved in commit 2184fbdb82f51e9e5f47d0c1fe8bfbcdf1a106cd

Reply:
Solved again in [3c12f1](https://github.com/mautic/mautic/pull/12379/commits/a3c12f1f0fa0dcbdf6f85f222fe84188b2203134)